python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -842,13 +841,16 @@ async def resolve_runOrError(self, graphene_info: ResolveInfo, runId):
     def resolve_runsFeedOrError(
         self,
         graphene_info: ResolveInfo,
-        limit: int,
+        limit: Optional[int] = None,
```

Comment:
Why make this optional?

Reply:
yeah, your idea is right. basically it didn't make sense to have limit (or cursor, but that was already optional) for getting the count. but it also felt gross the make the limit optional and have a runtime error if it's not passed to the resolver. idk what the right pattern to do here is or if we've had similar situations in the past to reference

Reply:
a separate object is appealing though

Reply:
given the weirdness introduced, I think I would prefer a solution where this count is a separate query -- I believe you can bundle together multiple queries into the same request, so having these be independent feels like the right call, e.g.

```
query RunsFeedEntryQuery($cursor: String, $limit: Int!, $filter: RunsFilter) {
  runsFeedOrError(cursor: $cursor, limit: $limit, filter: $filter) { ... }
  runsCountOrError(filter: $filter) {...}
}
```

Reply:
yeah i went ahead and started it here https://github.com/dagster-io/dagster/pull/24813